src/Speckle.Objects/Interfaces.cs

```diff
@@ -126,7 +126,7 @@ public interface IProperties : ISpeckleObject
   Dictionary<string, object?> properties { get; }
 }
 
-public interface IDataObject : ISpeckleObject, IProperties, IDisplayValue<IReadOnlyList<Base>>
+public interface IDataObject : IProperties, IDisplayValue<IReadOnlyList<Base>>
```

Comment:
Talking out loud: we're removing `ISpeckleObject` because the concrete implementation inherits from base, so we're bringing in those anyway via that route; we only want to enforce the properties.

Reply:
IProperties inherits from ISpeckleObject so it's redundant to leave it there.